sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -6719,6 +6719,12 @@ public void testMinMaxAvgDailyCountWithLimit() throws Exception
                             )
                         )
                         .setInterval(querySegmentSpec(Filtration.eternity()))
+                        .setLimitSpec(
```

Comment:
I am surprised that `explanation` didn't need to change here. The object should also have the same limit spec as this native plan.

Reply:
Yes the previous would have been `limitSpec=NoopLimitSpec` but now it changed to `limitSpec=DefaultLimitSpec{columns='[]', offset=0, limit=1}` . But considering the query has a LIMIT 1 at the end, I thought this was acceptable. I updated the limit to a different value and it shows up in the DefaultLimitSpec